Revision en docs (#1323)
mercadopago/devsite-docs: guides/online-payments/checkout-api/receiving-payment-by-card-core-methods.en.md

>
> This documentation uses the new library version. To see the previous version, go to [integrate payment for cards with MercadoPago.js V1 section](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/online-payments/checkout-api/v1/receiving-payment-by-card).

### 1. Include MercadoPago.js library

> **Use our official library to access Mercado Pago API** from your application and collect data securely.

Reviewer: ```suggestion
Use our **official library** to access Mercado Pago API from your application and collect data securely.
```

>

> > Consider that the form should be available before all the following steps for correct operation.

Reviewer: ```suggestion
> Before following the steps below, make sure that the form is available, so it can operate correctly.
```

------------

#### Get card payment method

> Avoid mistakes and offer the correct available installments by validating your customers' data as they fill it out. Use the code in the following example to identify payment method with the first 6 digits of the card.

Reviewer: ```suggestion
Avoid mistakes and offer the correct available installments by validating your customer's data as they fill it out. Use the code in the following example to identify payment method with the first 6 digits of the card.
```

}
```

### 5. Create card token

> Before payment submission, you must create a token containing all card information securely. You should generate it as follows:

Reviewer: ```suggestion
Before submitting the payment, you must create a token containing all card information securely. You should generate it as follows:
```
